Documentation/ceph-disaster-recovery.md

```diff
@@ -294,7 +294,7 @@ Assuming `dataHostPathData` is `/var/lib/rook`, and the `CephCluster` trying to
         container# monmaptool --rm c monmap  # Repeat this pattern until all the old ceph-mons are removed
         container# monmaptool --rm d monmap
         container# monmaptool --rm e monmap
-        container# monmaptool --add a 10.77.2.216:6789 monmap   # Replace it with the rook-ceph-mon-a address you got from previous command.
+        container# monmaptool --addv a [v2:10.77.2.216:3300,v1:10.77.2.216:6789] monmap   # Replace it with the rook-ceph-mon-a address you got from previous command.
```

Comment:
Did you intend to change to `--addv`?
```suggestion
        container# monmaptool --add a [v2:10.77.2.216:3300,v1:10.77.2.216:6789] monmap   # Replace it with the rook-ceph-mon-a address you got from previous command.
```

Reply:
I am a newbie of ceph.

I just think 'add' command can only add one ip-port pair, and addv (probably, add vector?) can enter multiple protocols together. The requested change is not executed in my experiment.

If I wrong, please point it out. (I really don't know about ceph)

Reply:
`--addv` is correct and was introduced to support the brackets, when the mons started to listen on the "official" port.